### PR TITLE
For issue: #18318 Added Content Description for logins_item.xml

### PR DESCRIPTION
**Pull request by priyanka0906**
fixes : #18318
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Comment by priyanka0906** [comment]
@eliserichard please take a look.

File: app/src/main/res/values/strings.xml
Comment (gabrielluong) [inline app/src/main/res/values/strings.xml:1688]: nit: Missing space after `<!--` and we typically describe the UI element when in the comments instead of using their IDs.

```suggestion
    <!-- Content Description for a login's web address text field -->
```

Comment (gabrielluong) [inline app/src/main/res/values/strings.xml:1688]: We also try to group the strings close to their related peers. Since this relates to logins, you can find more login related strings around https://github.com/mozilla-mobile/fenix/blob/cc0fdd1109d1ac7075505363dd737fc9af05811d/app/src/main/res/values/strings.xml#L1507. I would move these strings there and add an appropriate string prefix depending on the screen (eg `saved_logins` or `logins`).

File: app/src/main/res/layout/logins_item.xml
Comment (gabrielluong) [inline app/src/main/res/layout/logins_item.xml:39]: Consider running the format on Android Studio, I think that should help reorder the XML attributes and format the attributes nicely. Otherwise, we would like to group the `android` attributes with the other `android` attributes.

Comment (eliserichards) [reply app/src/main/res/layout/logins_item.xml:39]: I believe the auto-format commands are CTRL+ALT+L on Windows or command+option+L on Mac

File: app/src/main/res/values/strings.xml
Comment (priyanka0906) [reply app/src/main/res/values/strings.xml]: @gabrielluong I have made the changes

Comment (priyanka0906) [reply app/src/main/res/values/strings.xml]: Done

File: app/src/main/res/layout/logins_item.xml
Comment (priyanka0906) [reply app/src/main/res/layout/logins_item.xml]: done

File: app/src/main/res/layout/logins_item.xml
Comment (gabrielluong) [inline app/src/main/res/layout/logins_item.xml:1]: Let's undo this. This is unfortunate one of the problem with the autoformat.

Comment (priyanka0906) [reply app/src/main/res/layout/logins_item.xml]: done

File: app/src/main/res/values/strings.xml
Comment (gabrielluong) [inline app/src/main/res/values/strings.xml:1504]: minor nit for the 2 comments here. Can we use the lower case "Description" to be consistent?
 
```suggestion
    <!-- Content description for a login's web address text field -->
```

**Review by gabrielluong — APPROVED** [review]

File: app/src/main/res/values/strings.xml
Comment (gabrielluong) [inline app/src/main/res/values/strings.xml:1505]:
```suggestion
    <string name="text_field_for_displaying_web_address">Text field for displaying the web address</string>
```

File: app/src/main/res/values/strings.xml
Comment (gabrielluong) [inline app/src/main/res/values/strings.xml:1507]:
```suggestion
    <string name="text_field_for_displaying_username">Text field for displaying the username</string>
```

File: app/src/main/res/values/strings.xml
Comment (priyanka0906) [reply app/src/main/res/values/strings.xml]: done

File: app/src/main/res/values/strings.xml
Comment (priyanka0906) [reply app/src/main/res/values/strings.xml]: done

File: app/src/main/res/values/strings.xml
Comment (priyanka0906) [reply app/src/main/res/values/strings.xml]: done

**Comment by gabrielluong** [comment]
@priyanka0906 Thank you for the contribution! I don't see any additional changes needed here. We will wait for CI to do pass before landing this.

**Review by mcarare — CHANGES_REQUESTED** [review]
@eliserichards What is this PR trying to achieve? This breaks a11y for the saved logins screen. The login item is already announced with the web address and username for that page. This PR would break this and announce the same text for all saved logins items.

**Comment by gabrielluong** [comment]
> @eliserichards I am wondering what is this PR trying to achieve? This breaks a11y for the saved logins screen. The login item is already announced with the web address and username for that page. This PR would break this and announce the same text for all saved logins items.

Thank you for the catch.

**Comment by priyanka0906** [comment]
> @eliserichards I am wondering what is this PR trying to achieve? This breaks a11y for the saved logins screen. The login item is already announced with the web address and username for that page. This PR would break this and announce the same text for all saved logins items.



> @eliserichards I am wondering what is this PR trying to achieve? This breaks a11y for the saved logins screen. The login item is already announced with the web address and username for that page. This PR would break this and announce the same text for all saved logins items.

@mcarare  right..it will announce the same text for the same logins items...earlier it was reading the web address and username...I have asked about this to @eliserichards before starting work on this.
Is this issue invalid ?

**Comment by mcarare** [comment]
Closing this as the issue is invalid.

**Pull request closed without merging**
